packages/dds/merge-tree/src/test/revertibleFarm.spec.ts

```diff
@@ -35,14 +36,14 @@ const defaultOptions = {
 };
 
 describe("MergeTree.Client", () => {
-	doOverRanges(defaultOptions, ({ minLength: minLen, revertOps: opsWithRevert, revertOps }) => {
+	doOverRanges(defaultOptions, ({ minLength: minLen, concurrentOpsWithRevert, revertOps }) => {
```

Comment:
Missing revertOps property name? I'm not sure if that's intentional or if it would even affect anything, but figured I would point it out.

Reply:
it was broken before, it was setting opsWithRevert = revertOps, which is incorrect. this fixes it, and removes the funky syntax to re-maps the name